integration/flink/flink-stateful-application/README.md

```diff
@@ -0,0 +1,52 @@
+# OpenLineage - Flink sample stateful application

```

Comment:
_Minor_: We can make this a bit more clear that this is an flink example. I would move the example code under `examples/`:

```
integration/
   examples/
     stateful/
```

 As a reference for the directory structure, see [`proxy/examples/kafka`](https://github.com/OpenLineage/OpenLineage/tree/main/proxy/examples/kafka)